### ISSUE TITLE
Wrong label for clustered standard errors of fixest model

### ISSUE THREAD

**Issue by turbanisch** [vincentarelbundock/modelsummary#478]
I think I found another (very) minor inconsistency, again relating to standard error settings of `fixest` models. In the example below, the model contains standard errors that are clustered by the first variable used for fixed effects, `Product`. This is confirmed by `etable()`. When I display the regression output with `modelsummary()` without specifying the `vcov` argument, the reported standard errors are clustered ones but the last row still says "IID". 

As I understand the help file, `modelsummary()` takes the default standard errors of the model object (in this case clustered) if no `vcov` is specified. So apparently only the "IID" label is an issue here. 

I was also wondering how I can modify the row label for the standard errors (currently abbreviated as "Std.Errors"), for example if I wanted to squeeze in the fact that standard errors are clustered (e.g., "S.E.: Clustered"). Apparently, `gof_map = NA` removes the row on standard errors but there seems to be no way to define its label in the output by providing a dataframe to the `gof_map` argument.

Thanks! I really wish the `fixest` authors would embrace `modelsummary()` instead of their own `etable()` and ensure that everything works as expected! 🤓

``` r
library(fixest)
library(modelsummary)

mod_trade <- feols(Euros ~ dist_km | Product + Destination + Origin, data = trade)

# model default is clustered by Product
etable(mod_trade)
#>                              mod_trade
#> Dependent Var.:                  Euros
#>                                       
#> dist_km         -47,599.1*** (7,328.6)
#> Fixed-Effects:  ----------------------
#> Product                            Yes
#> Destination                        Yes
#> Origin                             Yes
#> _______________ ______________________
#> S.E.: Clustered            by: Product
#> Observations                    38,325
#> R2                             0.28482
#> Within R2                      0.03063
#> ---
#> Signif. codes: 0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

# vcov unspecified: returns the SE for ~Product but shows "IID"
modelsummary(mod_trade, 
             gof_omit = "^(?!Std)",
             output = "markdown")
```

|            |  Model 1   |
|:-----------|:----------:|
| dist_km    | -47599.054 |
|            | (7328.605) |
| Std.Errors |    IID     |

``` r
# vcov = "iid": correct output 
modelsummary(mod_trade, 
             gof_omit = "^(?!Std)",
             output = "markdown",
             vcov = "iid")
```

|            |  Model 1   |
|:-----------|:----------:|
| dist_km    | -47599.054 |
|            | (1368.735) |
| Std.Errors |    IID     |

``` r
# vcov clustered: correct output 
modelsummary(mod_trade, 
             gof_omit = "^(?!Std)",
             output = "markdown",
             vcov = ~Product)
```

|            |   Model 1   |
|:-----------|:-----------:|
| dist_km    | -47599.054  |
|            | (7328.605)  |
| Std.Errors | by: Product |

<sup>Created on 2022-05-21 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

<details style="margin-bottom:10px;">
<summary>
Session info
</summary>

``` r
sessioninfo::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.1.2 (2021-11-01)
#>  os       macOS Monterey 12.3.1
#>  system   aarch64, darwin20
#>  ui       X11
#>  language (EN)
#>  collate  en_US.UTF-8
#>  ctype    en_US.UTF-8
#>  tz       Europe/Berlin
#>  date     2022-05-21
#>  pandoc   2.17.1.1 @ /Applications/RStudio.app/Contents/MacOS/quarto/bin/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package      * version     date (UTC) lib source
#>  backports      1.4.1       2021-12-13 [1] CRAN (R 4.1.1)
#>  bayestestR     0.12.1      2022-05-02 [1] CRAN (R 4.1.1)
#>  checkmate      2.1.0       2022-04-21 [1] CRAN (R 4.1.1)
#>  cli            3.3.0       2022-04-25 [1] CRAN (R 4.1.1)
#>  codetools      0.2-18      2020-11-04 [1] CRAN (R 4.1.2)
#>  colorspace     2.0-3       2022-02-21 [1] CRAN (R 4.1.1)
#>  crayon         1.5.1       2022-03-26 [1] CRAN (R 4.1.1)
#>  datawizard     0.4.1.1     2022-05-21 [1] Github (easystats/datawizard@c9aad90)
#>  digest         0.6.29      2021-12-01 [1] CRAN (R 4.1.1)
#>  dreamerr       1.2.3       2020-12-05 [1] CRAN (R 4.1.0)
#>  ellipsis       0.3.2       2021-04-29 [1] CRAN (R 4.1.0)
#>  evaluate       0.15        2022-02-18 [1] CRAN (R 4.1.1)
#>  fansi          1.0.3       2022-03-24 [1] CRAN (R 4.1.1)
#>  fastmap        1.1.0       2021-01-25 [1] CRAN (R 4.1.0)
#>  fixest       * 0.10.5      2022-05-21 [1] https://fastverse.r-universe.dev (R 4.1.2)
#>  Formula        1.2-4       2020-10-16 [1] CRAN (R 4.1.0)
#>  fs             1.5.2       2021-12-08 [1] CRAN (R 4.1.1)
#>  future         1.25.0      2022-04-24 [1] CRAN (R 4.1.1)
#>  future.apply   1.9.0       2022-04-25 [1] CRAN (R 4.1.1)
#>  generics       0.1.2       2022-01-31 [1] CRAN (R 4.1.1)
#>  globals        0.15.0      2022-05-09 [1] CRAN (R 4.1.1)
#>  glue           1.6.2       2022-02-24 [1] CRAN (R 4.1.1)
#>  highr          0.9         2021-04-16 [1] CRAN (R 4.1.0)
#>  htmltools      0.5.2       2021-08-25 [1] CRAN (R 4.1.1)
#>  httr           1.4.3       2022-05-04 [1] CRAN (R 4.1.1)
#>  insight        0.17.1      2022-05-13 [1] CRAN (R 4.1.1)
#>  kableExtra     1.3.4       2021-02-20 [1] CRAN (R 4.1.0)
#>  knitr          1.39        2022-04-26 [1] CRAN (R 4.1.1)
#>  lattice        0.20-45     2021-09-22 [1] CRAN (R 4.1.2)
#>  lifecycle      1.0.1       2021-09-24 [1] CRAN (R 4.1.1)
#>  listenv        0.8.0       2019-12-05 [1] CRAN (R 4.1.0)
#>  lmtest         0.9-40      2022-03-21 [1] CRAN (R 4.1.1)
#>  magrittr       2.0.3       2022-03-30 [1] CRAN (R 4.1.1)
#>  modelsummary * 0.10.0.9000 2022-05-21 [1] Github (vincentarelbundock/modelsummary@5a46d25)
#>  munsell        0.5.0       2018-06-12 [1] CRAN (R 4.1.0)
#>  nlme           3.1-157     2022-03-25 [1] CRAN (R 4.1.1)
#>  numDeriv       2016.8-1.1  2019-06-06 [1] CRAN (R 4.1.0)
#>  parallelly     1.31.1      2022-04-22 [1] CRAN (R 4.1.1)
#>  parameters     0.17.0      2022-03-10 [1] CRAN (R 4.1.1)
#>  performance    0.9.0.6     2022-05-21 [1] Github (easystats/performance@445af86)
#>  pillar         1.7.0       2022-02-01 [1] CRAN (R 4.1.1)
#>  pkgconfig      2.0.3       2019-09-22 [1] CRAN (R 4.1.0)
#>  purrr          0.3.4       2020-04-17 [1] CRAN (R 4.1.0)
#>  R.cache        0.15.0      2021-04-30 [1] CRAN (R 4.1.0)
#>  R.methodsS3    1.8.1       2020-08-26 [1] CRAN (R 4.1.0)
#>  R.oo           1.24.0      2020-08-26 [1] CRAN (R 4.1.0)
#>  R.utils        2.11.0      2021-09-26 [1] CRAN (R 4.1.1)
#>  R6             2.5.1       2021-08-19 [1] CRAN (R 4.1.1)
#>  Rcpp           1.0.8.3     2022-03-17 [1] CRAN (R 4.1.1)
#>  reprex         2.0.1       2021-08-05 [1] CRAN (R 4.1.1)
#>  rlang          1.0.2       2022-03-04 [1] CRAN (R 4.1.1)
#>  rmarkdown      2.14        2022-04-25 [1] CRAN (R 4.1.1)
#>  rstudioapi     0.13        2020-11-12 [1] CRAN (R 4.1.0)
#>  rvest          1.0.2       2021-10-16 [1] CRAN (R 4.1.1)
#>  sandwich       3.0-1       2021-05-18 [1] CRAN (R 4.1.0)
#>  scales         1.2.0       2022-04-13 [1] CRAN (R 4.1.1)
#>  sessioninfo    1.2.2       2021-12-06 [1] CRAN (R 4.1.1)
#>  stringi        1.7.6       2021-11-29 [1] CRAN (R 4.1.1)
#>  stringr        1.4.0       2019-02-10 [1] CRAN (R 4.1.0)
#>  styler         1.7.0       2022-03-13 [1] CRAN (R 4.1.1)
#>  svglite        2.1.0       2022-02-03 [1] CRAN (R 4.1.1)
#>  systemfonts    1.0.4       2022-02-11 [1] CRAN (R 4.1.1)
#>  tables         0.9.6       2020-09-22 [1] CRAN (R 4.1.0)
#>  tibble         3.1.7       2022-05-03 [1] CRAN (R 4.1.1)
#>  utf8           1.2.2       2021-07-24 [1] CRAN (R 4.1.0)
#>  vctrs          0.4.1       2022-04-13 [1] CRAN (R 4.1.1)
#>  viridisLite    0.4.0       2021-04-13 [1] CRAN (R 4.1.0)
#>  webshot        0.5.3       2022-04-14 [1] CRAN (R 4.1.1)
#>  withr          2.5.0       2022-03-03 [1] CRAN (R 4.1.1)
#>  xfun           0.31        2022-05-10 [1] CRAN (R 4.1.1)
#>  xml2           1.3.3       2021-11-30 [1] CRAN (R 4.1.1)
#>  yaml           2.3.5       2022-02-21 [1] CRAN (R 4.1.1)
#>  zoo            1.8-10      2022-04-15 [1] CRAN (R 4.1.1)
#> 
#>  [1] /Library/Frameworks/R.framework/Versions/4.1-arm64/Resources/library
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

</details>

**Comment by turbanisch** [comment]
Sorry for the last part, I found it already:

```r
gm <- tibble(
  raw = "vcov.type",
  clean = "S.E.: clustered",
  fmt = 0
)
```

**Comment by vincentarelbundock** [comment]
Thanks for the report. I opened an issue upstream: https://github.com/lrberge/fixest/issues/313

**Comment by turbanisch** [comment]
Thanks!

**Comment by vincentarelbundock** [comment]
Oh, and just one editorial comment: I think it's awesome that we can have `etable`. Obviously, I like `modelsummary`, but it is super heavy in terms of dependencies. So having a one-package solution is pretty great in some applications 

:smile:

**Comment by turbanisch** [comment]
That's certainly true! I find myself using `fixest` almost exclusively these days, yet I prefer the `modelsummary` interface and the option to produce HTML output. The only thing I struggle with on a regular basis is displaying 2SLS results with their associated statistics 😬

**Comment by vincentarelbundock** [comment]
Fixed in d41f3be7c11, thanks to the information in https://github.com/lrberge/fixest/issues/313

Thanks for the report!
